.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*]
+charset = "utf-8"
+
+[*.coffee]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
```

Comment:
Okay- But having it will make it a bit easier for people to automatically adhere to your code style guidelines.